code/modules/reagents/newchem/other.dm

```diff
@@ -117,6 +117,12 @@ datum/reagent/colorful_reagent
 	result_amount = 6
 	mix_message = "The substance flashes multiple colors and emits the smell of a pocket protector."
 
+datum/reagent/colorful_reagent/reaction_mob(var/mob/living/simple_animal/M, var/method=TOUCH, var/volume)
+    if(M)
```

Comment:
So, this doesn't actually check to make sure that M is, indeed, a simple animal. Which means all mobs can be colored. Like it was before that was explicitly removed, because coloring humanoids screws up HUDs. Whoops!